src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -465,25 +465,25 @@ static bool isElementNameForbidden(string name)
                         return false;
                 }
             }
+        }
 
-            // Returns 3 for "Item3".
-            // Returns -1 otherwise.
-            static int matchesCanonicalElementName(string name)
+        // Returns 3 for "Item3".
+        // Returns -1 otherwise.
+        internal static int MatchesCanonicalTupleElementName(string name)
+        {
+            if (name.StartsWith("Item", StringComparison.Ordinal))
             {
-                if (name.StartsWith("Item", StringComparison.Ordinal))
+                string tail = name.Substring(4);
```

Comment:
```suggestion
                string tail = name.Substring("Item".Length);
```

(will be the exact same jitted code, but this helps make it clearer what's going on (imo)).

Reply:
> will be the exact same jitted code

It looks like `get_Length()` is called.